src/projects/pull-request/pull-request-page.html

```diff
@@ -171,7 +171,8 @@
         diff: String,
         selectedPull: {
           type: Object,
-          observer: '_fetchAjax'
+          observer: '_fetchAjax',
+          notify: true
```

Comment:
I think this is the correct way, but this will likely break.other stuff. I.had reported the issue to fix this and would like to do the changes all at once. Can this be reverted so it will be fixed alltogether when working on the other issue?